.github/actions/python-black-lint/action.yml

```diff
@@ -0,0 +1,17 @@
+# Define action name, description, and inputs
+name: "Python Black Lint"
+description: "Lint python code with black"
+
+# Define action steps
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-python@v2
+      with:
+        python-version: "3.x"
+        architecture: x64
+
+    - run: |
+        pip install black
+        black --check .
+      shell: bash
```

Comment:
Let's split the job into separate "run" keys, there's not benefit in keeping this in one (unlike Dockerfile where each layer is cached and costs disk space) and it's easier to catch which step failed.